frontend/src/metabase/dashboard/components/DashboardActions.jsx

```diff
@@ -54,15 +53,12 @@ export const getDashboardActions = (
     if (canCreateSubscription && !isFullscreen) {
       buttons.push(
         <Tooltip tooltip={t`Subscriptions`} key="dashboard-subscriptions">
-          <span>
-            <DashboardHeaderButton
-              disabled={!canManageSubscriptions}
-              onClick={onSharingClick}
-              data-metabase-event="Dashboard;Subscriptions"
-            >
-              <Icon size={18} name="subscription" />
-            </DashboardHeaderButton>
-          </span>
+          <DashboardHeaderButton
```

Comment:
Remove span and change the underlying implementation of `DashboardHeaderButton` to base off `Button` component which supports <kbd>Tab</kbd> selection